### PR TITLE
Node e2e test - propagate errors from tests to runner exit code.

### PR DESCRIPTION
**Pull request by pwittrock**
Have the runner exit(1) if any of the tests fail

### CODE REVIEW

File: test/e2e_node/runner/run_e2e.go [hunk 1]
```diff
@@ -96,9 +96,19 @@ func main() {
...
+	errCount := 0
+	for i := 0; i < len(hs); i++ {
+		if <-results != nil {
+			errCount++
+		}
+	}
+
+	// Set the exit code if there were failures
+	if errCount > 0 {
+		glog.Infof("Failure: %d errors encountered.", errCount)
```
Comment (vishh) [inline test/e2e_node/runner/run_e2e.go]: nit: Shouldn't this be glog.Errorf(...)?

Comment (pwittrock) [inline test/e2e_node/runner/run_e2e.go]: Yup.  Done

**Comment by vishh** [comment]
LGTM

**Comment by pwittrock** [comment]
@k8s-bot test this please

**Pull request merged**